### ISSUE TITLE
 Query Builder is not supported by MongoDB怎么处理呀！

### ISSUE THREAD

**Issue by yulintianxia** [typeorm/typeorm#3492]
、 Query Builder is not supported by MongoDB怎么处理呀！如果我要使用typeorm查询一定范围的数据

**Comment by pleerock** [comment]
Query Builder is for relational database. For mongodb you need to create a cursor (there is a method in repository and entity manager).

Please create issues in plain english. I don't know Chinese.

**Comment by vip30** [comment]
Here is the example of retrieve data  
```typescript
repository.find(<entity-name>, {
      where: {
        timestamp: {$lt: 150000},
      },
    })
```

You can find the example by clicking [here](https://github.com/typeorm/typeorm/blob/master/docs/mongodb.md)

**Comment by pleerock** [comment]
I'll close this issue and going to block @yulintianxia in the case if he'll continue to spam issues.

**Comment by pairmix** [comment]
Hi there,
I understand we have to use createCursor/createEntityCursor instead of createQueryBuilder.
But where can we find examples how to use it?

https://github.com/typeorm/typeorm/blob/master/docs/mongodb.md
